Automattic/jetpack
# Akismet: update Jetpack replacement logo styles

## Proposed changes:

A couple of years ago, we added code to replace the Akismet logo with the Jetpack logo in the Akismet plugin. This meant that the Jetpack logo would display instead if the user had a Jetpack subscription.

<img width="810" alt="Screenshot 2023-06-20 at 16 37 00" src="https://github.com/Automattic/jetpack/assets/17325/8f3662d2-30ed-438d-a55f-3daef7a96441">

The product at the time looked a little different:
* It was named "Jetpack Anti-Spam" with no reference to Akismet
* There was a plan to build separate configuration tools for Anti-Spam in preference to using the Akismet plugin

In 2023, we call the product "Jetpack Akismet Anti Spam" officially, so we're upfront that Akismet is powering this service. As far as I know we also have no plans to build a separate admin tool for Anti-Spam.

This PR starts showing the Jetpack Akismet Anti-Spam logo to customers of that product, rather than the generic Jetpack logo.

<img width="1244" alt="Screenshot 2023-07-06 at 3 32 44 PM" src="https://github.com/Automattic/jetpack/assets/17325/088b6925-41e4-426b-9a24-fa09dc701c07">

### Other information:


## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:
On a Jetpack development site, try signing up for a Jetpack Anti-Spam subscription and then visit `/wp-admin/admin.php?page=akismet-key-config`. You should always see the Jetpack Akismet Anti-Spam logo in the masthead.

On a fresh Jetpack development site, don't connect Jetpack and try activating the Akismet plugin via the Plugins page and visiting `/wp-admin/admin.php?page=akismet-key-config`. You should see the normal Akismet logo.
